net/aria2/patches/0001-Logger-Fix-format-string-overflow-in-writeHeader.patch

```diff
@@ -0,0 +1,94 @@
+From 8718757e6c62aeffd81a2ed9fa88e0c05b6412cf Mon Sep 17 00:00:00 2001
+From: Alexander Egorenkov <egorenar-dev@posteo.net>
+Date: Sat, 7 Oct 2023 13:39:06 +0200
+Subject: [PATCH 1/1] Logger: Fix format string overflow in writeHeader()
```

Comment:
This patch needs to be upstreamed. I did not see that it was send to upstream.

Reply:
i will give it a try but it seems to be dead :/

Reply:
What? The project is not dead. See: https://github.com/aria2/aria2/commits/master

Reply:
hmm, ok, then i was confused by so many issues and pull requests w/o answer.